lms/djangoapps/branding/__init__.py

```diff
@@ -17,7 +17,7 @@
 from openedx.core.djangoapps.content.course_overviews.models import CourseOverview
 
 
-def get_visible_courses(org=None):
+def get_visible_courses(org=None, filter_=None):
```

Comment:
It's a python convention that allows you to distinguish it from any built-in names/functions, in this case from the built-in `filter` function.